### PR TITLE
Fix recipients alignment

### PR DESCRIPTION
**Pull request by sanketfirodiya**
Fixes https://github.com/superhuman/superhuman-ios/issues/435

### CODE REVIEW

File: VENTokenField/VENTokenField.m [hunk 1]
```diff
@@ -311,7 +311,12 @@ - (void)layoutToLabelInView:(UIView *)view origin:(CGPoint)origin currentX:(CGFl
     self.toLabel.frame = newFrame;
     
     [view addSubview:self.toLabel];
-    *currentX += self.toLabel.hidden ? CGRectGetMinX(self.toLabel.frame) : CGRectGetMaxX(self.toLabel.frame) + self.toLabelPadding;
+
+    if ([self.dataSource respondsToSelector:@selector(marginBetweenToFieldAndTokens)]) {
+        *currentX += [self.dataSource marginBetweenToFieldAndTokens];
+    } else {
+        *currentX += self.toLabel.hidden ? CGRectGetMinX(self.toLabel.frame) : CGRectGetMaxX(self.toLabel.frame) + self.toLabelPadding;
+    }
```
Comment (emuye) [inline VENTokenField/VENTokenField.m]: I think this should continue to use the existing format of a property on VENTokenField that can be overriden by the delegate or can be set like toLabel. I would think this code would stay the same but we'd add the margin in.

File: VENTokenField/VENTokenField.m [hunk 2]
```diff
@@ -338,6 +343,12 @@ - (void)layoutTokensWithCurrentX:(CGFloat *)currentX currentY:(CGFloat *)current
         
         [self.tokens addObject:token];
 
+        // we need this adjustment to horizontally align the tokens with the toLabel
+        if (i == 0) {
+            CGFloat toLabelMidY = self.toLabel.frame.origin.y + self.toLabel.frame.size.height/2;
+            *currentY = toLabelMidY - token.frame.size.height/2;
+        }
```
Comment (emuye) [inline VENTokenField/VENTokenField.m:349]: What is the normal currentY here? Just trying to understand why we need to make a code change to center the label.

**Review by emuye — CHANGES_REQUESTED** [review]

File: VENTokenField/VENTokenField.m (hunk 1 above)
Comment (sanketfirodiya) [reply VENTokenField/VENTokenField.m]: 👍

File: VENTokenField/VENTokenField.m (hunk 2 above)
Comment (sanketfirodiya) [reply VENTokenField/VENTokenField.m:349]: Before this change, it was setting the currentY to - 
`*currentY += token.frame.size.height;`
This makes sure we horizontally align the token to the label.

**Review by emuye — APPROVED** [review]

**Pull request merged**